### PR TITLE
Lra 432 avm write socket is alive method

### PR DESCRIPTION
**Pull request by britaumich**
I wrote a sockets_are_alive task and want to discuss it.
The idea is that this task will run as a cron job every SOCKET_ALIVE_PERIOD period of time. It will scan the redis database for <facility_id>_status keys and read their values (time of the pong messages from websockets).
If the difference between Time.now and those values is bigger than SOCKET_ALIVE_PERIOD seconds then the socket is not responding.

### CODE REVIEW

File: lib/tasks/sockets_are_alive.rake [hunk 1]
```diff
@@ -0,0 +1,38 @@
+desc "This will find websockets that are not responding"
+task sockets_are_alive: :environment do
+# run this task every SOCKET_ALIVE_PERIOD time
+  require 'redis'
+  redis = Redis.new(host: "localhost")
+  # @@api_logger ||= Logger.new("#{Rails.root}/log/sockets_are_alive.log")
+
+  # api_logger.debug "get access token for update_campus_list, error: No access_token - #{result['error']}"
```
Comment (rsmoke) [inline lib/tasks/sockets_are_alive.rake:8]: what is this for?

**Review by rsmoke — COMMENTED** [review]
I am having troubles following the logic. We can go over this together.

File: lib/tasks/sockets_are_alive.rake (hunk 1 above)
Comment (britaumich) [reply lib/tasks/sockets_are_alive.rake]: removed copied from classrooms code; add comment

**Comment by britaumich** [comment]
I added a task description to the file

**Comment by rsmoke** [comment]
Rita said close this branch

**Pull request closed without merging**